test/integration/emitter_factory_test.ts

```diff
@@ -107,7 +107,13 @@ function testEmitter(fn: TransportFunction, bodyAttr: string) {
   });
 }
 
-function assertBinary(response: Record<string, string>) {
+/**
+ * Verify the received binary answer compare to the original fixture message
+ *
+ * @param {Record<string, Record<string, string>>} response received to compare to fixture
+ * @return {void} void
+ */
+export function assertBinary(response: Record<string, string>): void {
```

Comment:
Sorry, this is not in scope with the PR.

Reply:
The export was added to be able to share some logics and avoid duplicated code

Reply:
> The export was added to be able to share some logics and avoid duplicated code

Makes sense to me.